Library/Formula/gcviewer.rb

```diff
@@ -0,0 +1,19 @@
+class Gcviewer < Formula
+  desc "Java garbage collection visualization tool"
+  homepage "https://github.com/chewiebug/GCViewer"
+  url "https://github.com/chewiebug/GCViewer/archive/1.34.1.tar.gz"
+  sha256 "e0e97a94c80be8323772dc8953d463fe8eaf60c3a1c5f212d0b575a3b5c640ba"
+
+  depends_on :java
+  depends_on "maven" => :build
+
+  def install
+    system "mvn", "-Dmaven.surefire.debug=-Duser.language=en", "clean", "install"
```

Comment:
I suspect if you're running `mvn` here you'll need to pass `ENV.java_cache` in `def install`.